test/uuid/ecto/type_config_test.exs

```diff
@@ -0,0 +1,143 @@
+defmodule UUID.Ecto.TypeConfigTest do
```

Comment:
given the module is private, I probably wouldn't test it directly and instead go through the public interface as mentioned in the previous comment. Now that we have it though, it doesn't hurt it keep it around but it might be additional maintenance burden.